src/test/java/org/omnifaces/test/resourcehandler/viewresourcehandler/ViewResourceHandlerIT.java

```diff
@@ -41,7 +41,7 @@ public class ViewResourceHandlerIT extends OmniFacesIT {
         + "<urlset xmlns=\"http://www.sitemaps.org/schemas/sitemap/0.9\">"
             + "<url><loc>{baseURL}entity.xhtml?id=1</loc><lastmod>2020-12-22T19:20:10Z</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url>"
             + "<url><loc>{baseURL}entity.xhtml?id=2</loc><lastmod>2020-12-22</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url>"
-            + "<url><loc>{baseURL}entity.xhtml?id=3</loc><lastmod>2020-12-22T15:20:10" + ZoneId.systemDefault().getRules().getOffset(Instant.now()) + "</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url>"
+            + "<url><loc>{baseURL}entity.xhtml?id=3</loc><lastmod>2020-12-22T15:20:10" + ZoneId.systemDefault().getRules().getOffset(LAST_MODIFIED.toLocalDateTime()) + "</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url>"
```

Comment:
Does `LocalDateTime.now()` work?

Reply:
sadly no, but If I understand correctly the problem is that 
we can't retrieve the offset of a current time and compare it
with the manually define LAST_MODIFIED, which has a 4 hours offset ...

Reply:
Strange. LocalDateTime isn't supposed to have any internal offset. Corretto thing again?

Reply:
At this point I'm going to try another jdk

Reply:
Will merge nonetheless. This change works fine for me as well. But first I want to analyze more things in relation to this.

Reply:
Right, it's a DST matter. DST is observed in Italy.

Reply:
ok! I suspected something about, because the were a diff of 1 hour...

Reply:
In fact I've tried also with Oracle 21.0.4 both `Instant` and `LocalDateTime` -> same error